emulated_hue/utils.py

```diff
@@ -72,9 +72,9 @@ def update_dict(dict1, dict2):
 
 def send_json_response(data) -> web.Response:
     """Send json response in unicode format instead of converting to ascii."""
-    return web.Response(
-        text=json.dumps(data, ensure_ascii=False), content_type="application/json"
-    )
+    response = bytearray(json.dumps(data, ensure_ascii=False), encoding="utf-8")
```

Comment:
Seems like this is not actually necessary. Restarting f.lux while its contantly pinging for a username will fix it and it will use the correct username. Maybe a bug with f.lux.

Reply:
I remember a user reporting issues with unicode characters otherwise.
So I'm starting to believe that the default utf-8 encoding doesn't work with Hue